audio/audio_driver.c

```diff
@@ -192,6 +192,12 @@ const char *config_get_audio_driver_options(void)
    return char_list_new_special(STRING_LIST_AUDIO_DRIVERS, NULL);
 }
 
+unsigned audio_driver_get_sample_size(void)
+{
+   audio_driver_state_t *audio_st = &audio_driver_st;
+   return (audio_st->flags & AUDIO_FLAG_USE_FLOAT) ? sizeof(float) : sizeof(int16_t);
```

Comment:
Best not create another useless wrapper function like this. Instead, just call audio_state_get_ptr() from outside to grab the pointer, then look at the 'flags' variable and just do the code manually that way. It's likely only going to be used once or twice in the entire codebase, waste of a function.

Reply:
I don't agree that it's useless; manually checking flags like that everywhere is error-prone. While developing this PR, I ran into bugs that were caused by exactly that kind of mistake. If you're concerned about performance, I could provide a macro or inline function instead.

That being said, I noticed that this function isn't used anyway, so I'll remove it.